### DISCUSSION TITLE
Simple feature requests

### DISCUSSION THREAD

**Discussion by Doze-D** [vgmtrans/vgmtrans#218] [General]
- Batch extract files into multiple separate folders within the selected location

- Make the program remember the last extraction location used

- Possible inclusion of a sample player within the program

Hope this is not too much asked, been using this tool since last year, but it has become painful to use because these features are missing. Otherwise it's excellent work!

**Comment by sykhro** [comment]
> Batch extract files into multiple separate folders within the selected location

As in, one folder per collection? Will add that to the list.

> Make the program remember the last extraction location used

This should already be happening. Or do you mean when invoking the program multiple times?


> Possible inclusion of a sample player within the program

This will be part of the Qt version. Legacy is already a mess as-is.

**Reply by Doze-D** [reply]
Yea, one folder per collection set. Right now I have one sample file opened that contains over 80 sets of samples, so it's understandably painful to extract those 80 sets separately and create separate folders for them. 

It won't happend on me for some reason, could be that the one I'm using is older version, might try to recompile newer version of it at some point (Win10 64x)

**Comment by sykhro** [comment]
> Batch extract files into multiple separate folders within the selected location

Will land in VGMTrans 1.1 (hopefully very soon)

> Make the program remember the last extraction location used

Implemented as of 3c75f981c2da96fdc65ca7df5f7de4aa6b5dc13a

> Possible inclusion of a sample player within the program

Planned for VGMTrans 1.2
